Cocoapods Example/PortalSwift/Portal.swift

```diff
@@ -157,10 +158,21 @@ class PortalWrapper {
     }
   }
 
-  func backup(backupMethod: BackupMethods.RawValue, user: UserResult, backupConfigs: BackupConfigs? = nil, completion: @escaping (Result<Bool>) -> Void) {
+  func backup(backupMethod: BackupMethods.RawValue, user: UserResult, backupConfigs: BackupConfigs? = nil, completion: @escaping (Result<String>) -> Void) {
     self.portal?.backupWallet(method: backupMethod, backupConfigs: backupConfigs) { (result: Result<String>) in
       guard result.error == nil else {
-        return completion(Result(error: result.error!))
+        do {
+          try self.portal?.api.storedClientBackupShare(success: false) { result in
+            guard result.error == nil else {
+              print("❌ handleBackup(): Error notifying Portal that backup share was not stored.")
+              return completion(Result(error: result.error!))
```

Comment:
To remove using `result.error!` you could do something like `guard let error = result.error else {` ... `Result(error: error)`